test/e2e/features/story_openshift.feature

```diff
@@ -56,14 +56,15 @@ Feature: 3 Openshift stories
 		# cleanup
 		And executing "oc delete project testproj-img" succeeds
 
-	@linux
-	Scenario: Create local image, push to registry, deploy (via podman login)
-		Given checking that CRC is running
-		And executing "podman pull quay.io/centos7/httpd-24-centos7" succeeds
-		When executing "oc new-project testproj-img" succeeds
-		And executing "podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.apps-crc.testing --tls-verify=false" succeeds
-		And executing "podman tag quay.io/centos7/httpd-24-centos7 default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test" succeeds
-		And executing "podman push default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test --tls-verify=false" succeeds
+	@darwin @linux @windows @local-img-to-registry @testdata @story_registry
+	Scenario: Pull image locally, push to registry, deploy
+		Given podman command is available
+		And executing "oc new-project testproj-img" succeeds
+		When pulling image "quay.io/centos7/httpd-24-centos7", logging in, and pushing local image to internal registry succeeds
```

Comment:
instead of this which is more messy why we can't just use `eval $(crc podman-env)` command direct from shell which will anyway add those env variables and then we should go as expected?